apps/platform/src/app/auth/otp/page.tsx

```diff
@@ -88,7 +89,28 @@ export default function AuthOTPPage(): React.JSX.Element {
       }
     }
   }
-
+  const handleResendOtp = async (userEmail: string): Promise<void> => {
+    try {
+      setIsLoadingRefresh(true)
+      const response = await fetch(
+        `${process.env.NEXT_PUBLIC_BACKEND_URL}/api/auth/resend-otp/${encodeURIComponent(userEmail)}`,
+        {
+          method: 'POST'
+        }
+      )
+      if (response.status === 429) {
+        toast.error("Couldn't send OTP, too many requests")
+        setIsLoadingRefresh(false)
+      } else if (response.ok)
+        toast.success('OTP successfully sent to your email')
+      setIsLoadingRefresh(false)
+    } catch (error) {
+      // eslint-disable-next-line no-console -- we need to log the error
+      console.error(`Failed to send OTP: ${error}`)
+      toast.error("Couldn't send OTP .")
+      setIsLoadingRefresh(false)
+    }
+  }
```

Comment:
Can you organize the API logic in `apps/platform/src/lib/api-functions` rest looks good

Reply:
Don't use your own calls to the API. Rather use this: https://github.com/keyshade-xyz/keyshade/blob/5884833fd7c36e0021ddc7e50d06fe3374e90c15/packages/api-client/src/controllers/user.ts#L55-L65

You can check the usage of these in any of the CLI files: https://github.com/keyshade-xyz/keyshade/tree/develop/apps/cli/src/commands

Reply:
> resendEmailChangeOTP

I got the point of the controller code from api-client but not the second thing you said
> You can check the usage of these in any of the CLI files: https://github.com/keyshade-xyz/keyshade/tree/develop/apps/cli/src/commands

Reply:
Oh, I meant that you can check any of the files in the `commands` directory to understand how to use these controllers.

Reply:
So i have to write a new controller for auth and utilise as a controller instance if I am getting it right? 

Is there a controller instance file similar instance to https://github.com/keyshade-xyz/keyshade/blob/develop/apps/cli/src/util/controller-instance.ts or I need to create one for the frontend of platform?